package.json

```diff
@@ -27,8 +27,8 @@
     ]
   },
   "dependencies": {
-    "@rsksmart/rif-relay-common": "1.0.0-alpha.1",
-    "@rsksmart/rif-relay-contracts": "1.0.0-alpha.0",
+    "@rsksmart/rif-relay-common": "github:rsksmart/rif-relay-common#PP-504/remove-worker-configuration",
+    "@rsksmart/rif-relay-contracts": "github:rsksmart/rif-relay-contracts#PP-504/remove-worker-configuration",
```

Comment:
Once we merge the related PRs, we need to update it

Reply:
updated